goreleaser.yml

```diff
@@ -163,6 +163,19 @@ docker_manifests:
     - 'ghcr.io/aquasecurity/trivy:{{ .Version }}-amd64'
     - 'ghcr.io/aquasecurity/trivy:{{ .Version }}-arm64'
   - name_template: 'public.ecr.aws/aquasecurity/trivy:latest'
-    image_templates: 
+    image_templates:
     - 'public.ecr.aws/aquasecurity/trivy:{{ .Version }}-amd64'
     - 'public.ecr.aws/aquasecurity/trivy:{{ .Version }}-arm64'
+
+signs:
+  - cmd: cosign
+    env:
+      - COSIGN_EXPERIMENTAL=1
+    certificate: '${artifact}.pem'
+    output: true
+    artifacts: checksum
+    args:
+      - sign-blob
```

Comment:
According to the document, what we need looks `docker_signs`. `sign-blob` is a subcommand for signing artifacts such as binaries and packages.
https://goreleaser.com/customization/docker_sign/

Am I missing something?

Reply:
Ah yes, you are totally right, this was my mistake, I'll fix that ASAP, really good catch, thank you for watching my back 🙋🏻‍♂️😍

Reply:
But we want to sign our binaries and packages as well as container images in another PR😉

Reply:
thank you, it should be fixed right now.